Sources_v3/StreamChat/Models/Attachment.swift

```diff
@@ -0,0 +1,310 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+public typealias ChatMessageAttachment = _ChatMessageAttachment<DefaultExtraData.Attachment>
+
+@dynamicMemberLookup
+public struct _ChatMessageAttachment<ExtraData: AttachmentExtraData>: Hashable {
+    /// A title.
+    public let title: String
+    /// An author.
+    public let author: String?
```

Comment:
nit: Once you figure out what all this means, please put it all in the documentation comments 😅

Reply:
@mantergo This is still valid. A lot of public-facing things here have non-descriptive documentation or no documentation at all.